packages/js-client/src/index.test.ts

```diff
@@ -180,7 +180,7 @@ test('Allow array query parameters', async () => {
   const siteId = uuidv4()
   const scope = nock(origin)
     .get(
-      `${pathPrefix}/sites/${siteId}/plugin_runs/latest?packages%5B%5D=%40scope%2Fpackage&packages%5B%5D=%40scope%2Fpackage-two`,
+      `${pathPrefix}/sites/${siteId}/plugin_runs/latest?packages%5B0%5D=%40scope%2Fpackage&packages%5B1%5D=%40scope%2Fpackage-two`,
```

Comment:
Original:
`plugin_runs/latest?packages%5B%5D=%40scope%2Fpackage&packages%5B%5D=%40scope%2Fpackage-two`
(decodes to `plugin_runs/latest?packages[]=@scope/package&packages[]=@scope/package-two`)

This PR:
`plugin_runs/latest?packages%5B0%5D=%40scope%2Fpackage&packages%5B1%5D=%40scope%2Fpackage-two`
(decodes to `plugin_runs/latest?packages[0]=@scope/package&packages[1]=@scope/package-two`)

The syntax used in this PR is equivalent and clearer